paper-autocomplete.html

```diff
@@ -128,7 +128,7 @@
     <div class="input-wrapper" role="combobox" aria-haspopup="true" aria-owns="suggestionsWrapper" aria-expanded$="[[_isSuggestionsOpened]]">
       <!-- For accessibility, it is needed to have a label or aria-label. Label is preferred -->
       <label for="autocompleteInput" class="sr-only">[[label]]</label>
-
+      <paper-input   name$="[[name]]" value="{{_selectedOption.value}}" hidden ></paper-input >
```

Comment:
can you explain what did you try to do here? I am not sure I follow. What is the purpose of this change?